File: Php/1046.php
<?php
[$inicio, $fin] = explode(" ", readline());

if ($inicio == $fin) {
    print("O JOGO DUROU 24 HORA(S)\n");
}
else if ($inicio > $fin) {
    print("O JOGO DUROU " . (24 - $inicio + $fin) . " HORA(S)\n");
}
else {
    print("O JOGO DUROU " . ($fin - $inicio) . " HORA(S)\n");
}
?>

File: Kotlin/1046.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    var inicio = input.nextInt();
    var fin = input.nextInt();
    
    if(inicio == fin){
        println("O JOGO DUROU 24 HORA(S)")
    }
    else if(inicio > fin){
        println("O JOGO DUROU ${24 - inicio + fin} HORA(S)")
    }
    else if(inicio < fin){
        println("O JOGO DUROU ${fin - inicio} HORA(S)")
    }
    
}
